app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,30 +58,33 @@ module SupportsFeatureMixin
-    :control                      => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
-    :cloud_tenant_mapping         => 'CloudTenant mapping',
-    :backup_create                => 'CloudVolume backup creation',
-    :backup_restore               => 'CloudVolume backup restore',
-    :cinder_service               => 'Cinder storage service',
-    :swift_service                => 'Swift storage service',
-    :delete                       => 'Deletion',
-    :disassociate_floating_ip     => 'Disassociate a Floating IP',
-    :discovery                    => 'Discovery of Managers for a Provider',
-    :evacuate                     => 'Evacuation',
-    :events                       => 'Query for events',
-    :launch_cockpit               => 'Launch Cockpit UI',
-    :live_migrate                 => 'Live Migration',
-    :migrate                      => 'Migration',
-    :provisioning                 => 'Provisioning',
-    :reboot_guest                 => 'Reboot Guest Operation',
-    :reconfigure                  => 'Reconfiguration',
-    :refresh_new_target           => 'Refresh non-existing record',
-    :regions                      => 'Regions of a Provider',
-    :resize                       => 'Resizing',
-    :retire                       => 'Retirement',
-    :smartstate_analysis          => 'Smartstate Analaysis',
-    :terminate                => 'Terminate a VM'
+    :associate_floating_ip    => 'Associate a Floating IP',
+    # FIXME: this is just a internal helper and should be refactored
+    :control                  => 'Basic control operations',
+    :cloud_tenant_mapping     => 'CloudTenant mapping',
+    :backup_create            => 'CloudVolume backup creation',
+    :backup_restore           => 'CloudVolume backup restore',
+    :cinder_service           => 'Cinder storage service',
+    :swift_service            => 'Swift storage service',
+    :delete                   => 'Deletion',
+    :disassociate_floating_ip => 'Disassociate a Floating IP',
+    :discovery                => 'Discovery of Managers for a Provider',
+    :evacuate                 => 'Evacuation',
+    :events                   => 'Query for events',
+    :launch_cockpit           => 'Launch Cockpit UI',
+    :live_migrate             => 'Live Migration',
+    :migrate                  => 'Migration',
+    :provisioning             => 'Provisioning',
+    :reboot_guest             => 'Reboot Guest Operation',
+    :reconfigure              => 'Reconfiguration',
+    :refresh_new_target       => 'Refresh non-existing record',
+    :regions                  => 'Regions of a Provider',
+    :resize                   => 'Resizing',
+    :retire                   => 'Retirement',
+    :smartstate_analysis      => 'Smartstate Analaysis',
+    :terminate                => 'Terminate a VM',
+    :reconfigure_vms          => 'Reconfigure Virtual Machines',
+    :snapshots                => 'Snapshots'
```

Comment:
could you leave the indentation level as is and move snapshots to it alphabetic position?